core/src/nucleus/ribosome/api/hash_entry.rs

```diff
@@ -15,6 +15,7 @@ pub fn get_entry_type(dna: &Dna, entry_type_name: &str) -> Result<EntryType, Opt
             holochain_core_types::error::RibosomeErrorCode::UnknownEntryType as i32,
         ))
     })?;
+
```

Comment:
Umm this stuff seems not relevant to this PR, but that's ok I guess...  :-P

Reply:
@zippy haha this is me triggering travis builds so i can inspect whether the cache changes worked... because you cannot tell until the next run 😅

Reply:
ah grr, the pain of travis...